src/components/side-panel/layer-panel/vis-config-by-field-selector.js

```diff
@@ -23,13 +23,12 @@ import PropTypes from 'prop-types';
 import {FormattedMessage, injectIntl} from 'react-intl';
 
 import {PanelLabel, PanelLabelWrapper, SidePanelSection} from 'components/common/styled-components';
-import FieldSelector from 'components/common/field-selector';
 import InfoHelperFacotry from 'components/common/info-helper';
 import DimensionScaleSelector from './dimension-scale-selector';
 import {camelize} from 'utils/utils';
+import FieldSelectorFactory from '../../common/field-selector';
 
-VisConfigByFieldSelectorFacotry.deps = [InfoHelperFacotry];
-export default function VisConfigByFieldSelectorFacotry(InfoHelper) {
+const VisConfigByFieldSelectorFactory = (InfoHelper, FieldSelector) => {
```

Comment:
same here, make it a function instead of const ...
put the deps definition on top of the function declaration for easy read.

Reply:
Done